internal/cmd/destinations.go

```diff
@@ -81,6 +81,7 @@ func newDestinationsRemoveCmd(cli *CLI) *cobra.Command {
 		Example: "$ infra destinations remove docker-desktop",
 		Args:    ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			forbiddenMsg := "You do not have privileges to disconnect destinations from infra; contact your admin\n\nRun `infra info` for more information about your session"
```

Comment:
This seems unnecessarily repetitive. Why not handle it once in `main.go`?

Reply:
Also suggest to avoid abbreviations. The latter part of the message also seems unnecessary

```
You do not have privileges for this action. Contact your administrator.
```

Reply:
i have a different problem with this. the message says "disconnect providers", but it's used everywhere for all kinds of endpoints. we probably want a unique message per endpoint

Reply:
updated